common/src/main/res/drawable/rounded_corner_grey_bgd.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/steel_gray_50" />
+    <corners android:radius="14dp" />
```

Comment:
Instead of adding a whole bunch of drawables for various backgrounds, you could use the `rounded_background.xml` with a theme, like this:

```
<View
...
android:background="@drawable/rounded_background"
android:theme="@style/SearchInputBackground"
```

in styles:

```
<style name="SearchInputBackground">
    <item name="backgroundColor">@color/gray_40</item>
    <item name="strokeColor">@color/gray_40</item>
    <item name="cornerRadius">8dp</item>
</style>
```